internal/ir/xds.go

```diff
@@ -206,6 +206,8 @@ type BackendWeights struct {
 type HTTPRoute struct {
 	// Name of the HTTPRoute
 	Name string `json:"name" yaml:"name"`
+	// Hostname that the route matches against
```

Comment:
since this is critical, lets enhance the `Validate` func to check for `len(Hostname) > 1`